matscipy/cauchy_born.py

```diff
@@ -0,0 +1,742 @@
+import numpy as np
+from ase.lattice.cubic import Diamond
+
+from ase.optimize import LBFGS
+
+from scipy.linalg import polar, sqrtm
+
+from itertools import permutations
+
+from sklearn.linear_model import Lasso, LinearRegression
+
+from scipy.stats.qmc import LatinHypercube, scale
```

Comment:
Currently matscipy does not depend on sklearn. Do we really need this? If so it should be an optional dependency: try to import it, and if it fails raise an exception later when someone tries to use the functionality.